docs/docs/benchmark.md

```diff
@@ -1,96 +1,262 @@
-  task = openml.tasks.get_task(task_id) # download the OpenML task
-  X, y = task.get_X_and_y() # get the data (not used in this example)
-  openml.config.apikey = 'FILL_IN_OPENML_API_KEY' # set the OpenML Api Key
-  # run classifier on splits (requires API key)
-  run = openml.runs.run_model_on_task(task,clf)
-  score = run.get_metric_score(sklearn.metrics.accuracy_score) # print accuracy score
-  print('Data set: %s; Accuracy: %0.2f' % (task.get_dataset().name,score.mean()))
-  run.publish() # publish the experiment on OpenML (optional)
-  print('URL for run: %s/run/%d' %(openml.config.server,run.run_id))
-```
-
-### Java (with WEKA)
-``` java
-public static void runTasksAndUpload() throws Exception {
-  OpenmlConnector openml = new OpenmlConnector();
-  // obtain the benchmark suite
-  Study benchmarksuite = openml.studyGet("OpenML100", "tasks");
-  Classifier tree = new REPTree(); // build a Weka classifier
-  for (Integer taskId : benchmarksuite.getTasks()) { // iterate over all tasks
-    Task t = openml.taskGet(taskId); // download the OpenML task
-    Instances d = InstancesHelper.getDatasetFromTask(openml, t); // obtain the dataset
-    openml.setApiKey("FILL_IN_OPENML_API_KEY");
-    int runId = RunOpenmlJob.executeTask(openml, new WekaConfig(), taskId, tree);
-    Run run = openml.runGet(runId);   // retrieve the uploaded run
-  }
-}
-```
-
-### R (with mlr)
-``` r
-library(OpenML)
-lrn = makeLearner('classif.rpart') # construct a simple CART classifier
-task.ids = getOMLStudy('OpenML100')$tasks$task.id # obtain the list of suggested tasks
-for (task.id in task.ids) { # iterate over all tasks
-  task = getOMLTask(task.id) # download single OML task
-  data = as.data.frame(task) # obtain raw data set
-  run = runTaskMlr(task, learner = lrn) # run constructed learner
-  setOMLConfig(apikey = 'FILL_IN_OPENML_API_KEY')
-  upload = uploadOMLRun(run) # upload and tag the run
-}
-```
 
 ## Creating new benchmark suites
 The set of datasets on OpenML.org can <a href="https://www.openml.org/new/data">easily be extended</a>, and additional OpenML benchmark suites,
 e.g., for regression and time-series data, can easily be created by defining sets of datasets according to specific needs.
-This is done by:
-<ul>
-<li><a href="new/study">Creating a new study</a> with the name of the benchmark as the alias.</li>
-<li>Adding new tasks to the study by <i>tagging</i> them with the name of the benchmark.</li>
-</ul>
+
+??? note "Python example"
+    ```python
+    
+    import openml
+    
+    # find 250 tasks that we are interested in, e.g., the tasks that have between
+    # 100 and 10000 instances and between 4 and 20 attributes
+    tasks = openml.tasks.list_tasks(number_instances='100..10000', number_features='4..20', size=250)
+    task_ids = list(tasks.keys())
+    
+    # create the benchmark suite
+    study = openml.study.create_benchmark_suite(None, "MidSize Suite", "illustrating how to create a benchmark suite", task_ids)
```

Comment:
I think it would be good to add a brief description of the arguments here. Even I don't know what they are and why the first one is `None`.